jupyterhub/orm.py

```diff
@@ -674,18 +675,29 @@ def new(
             orm_token.service = service
         if expires_in is not None:
             orm_token.expires_at = cls.now() + timedelta(seconds=expires_in)
+
         db.add(orm_token)
-        # load default roles if they haven't been initiated
-        # correct to have this here? otherwise some tests fail
         token_role = Role.find(db, 'token')
         if not token_role:
+            # FIXME: remove this.
+            # Creating a token before the db has roles defined should raise an error.
+            # PR #3460 should let us fix it by ensuring default roles are define
```

Comment:
#3460 should allow us to remove this by ensuring we run `init_role_creation` as part of intializing a db

Reply:
Yes!